enterprise/frontend/src/embedding-sdk/CHANGELOG.md

```diff
@@ -1,15 +1,3 @@
-## [0.1.6](https://github.com/metabase/metabase/compare/embedding-sdk-0.1.5...embedding-sdk-0.1.6) (2024-05-31)
```

Comment:
This is just for mocking if we're releasing from a different release branch/commit that doesn't have the same changelog content as master. Only the diff when creating a changelog should be apply to the PR targeting master.